tests/snowpark/test_snowpark_shared.py

```diff
@@ -0,0 +1,137 @@
+import logging
+from pathlib import Path
+from requirements.requirement import Requirement
+from unittest import mock
+from zipfile import ZipFile
+
+import pytest
+import typer
+
+import snowcli.cli.snowpark_shared as shared
+from snowcli.utils import SplitRequirements
+from tests.testing_utils.fixtures import *
+
+
+@mock.patch("tests.snowpark.test_snowpark_shared.shared.utils.parse_anaconda_packages")
+@mock.patch("tests.snowpark.test_snowpark_shared.shared.utils.install_packages")
+def test_snowpark_package(
+    mock_install, mock_parse, temp_dir, correct_requirements_txt, caplog
+):
+
+    mock_parse.return_value = SplitRequirements(
+        [], [Requirement.parse("totally-awesome-package")]
+    )
+
+    mock_install.return_value = (True, None)
+
+    with caplog.at_level(logging.INFO, logger="snowcli.cli.snowpark_shared"):
+        result = shared.snowpark_package("yes", False, "yes")
```

Comment:
I'm personally any assertions on logs unless necessary. First I don't see logs as "business logic", second it's often suggest bad design. 

In this case I don't think the assertion is necessary. If we want to be sure something was execute then let's mock it and do assertion on the mock.

Reply:
Done